build_web_compilers/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Update to run DDC in kernel mode, and consume kernel outlines instead of
   analyzer summaries.
 - Increased the upper bound for `package:analyzer` to `<0.37.0`.
+- Removed support for `build_root_app_summary` configuration option.
```

Comment:
We should mention that you can't build using analyzer summaries any more as well (technically the builder was public)

Reply:
Done